.azure-pipelines.yml

```diff
@@ -653,9 +677,9 @@ jobs:
       sudo snap install --classic --dangerous $(Pipeline.Workspace)/linux_amd64_snap/parsec*.snap
       parsec.cli --version
       # Make sure snap contains core but not backend
-      NOT_AVAILABLES=$(parsec.cli --help | grep -i "not available")
-      test "$(echo $NOT_AVAILABLES | grep -i 'backend')"
-      test -z "$(echo $NOT_AVAILABLES | grep -i 'core')"
+      NOT_AVAILABLE=$(parsec.cli --help | grep -i "not available")
```

Comment:
the `s` is here given it refers to modules that could be not available (so it's not a boolean information)

Reply:
`available` is an adjective, it don't take a 's' on plurial.

maybe `MODULES_NOT_AVAILABLE` sound better ?

Reply:
done in a8ae932